x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/Querier.java

```diff
@@ -156,7 +156,7 @@ public static SearchRequest prepareRequest(Client client, SearchSourceBuilder so
             String... indices) {
         source.timeout(timeout);
 
-        SearchRequest searchRequest = new SearchRequest(SWITCH_TO_FIELDS_API_VERSION);
+        SearchRequest searchRequest = new SearchRequest(MISSING_ORDER_IN_COMPOSITE_AGGS_VERSION);
```

Comment:
Not sure wether this is necessary, in theory there is only a few queries that need `missing_order`, most queries would still succeed on a node without this capability.

Reply:
I think you need to test this behavior in `SqlSearchIT`.
Why would a query succeed on a node without [this feature](https://github.com/elastic/elasticsearch/pull/76740)? Wouldn't there be an error about an unrecognized `missing_order` parameter?